jme3-core/src/main/java/com/jme3/scene/Spatial.java

```diff
@@ -1064,7 +1064,7 @@ public void setLocalRotation(Matrix3f rotation) {
      * <code>setLocalRotation</code> sets the local rotation of this node.
      *
      * @param quaternion
-     *            the new local rotation.
+     *            the new local rotation (not null, norm=1 or nearly so)
```

Comment:
Hello, I think the word 'nearly so' needs some elaboration, does that mean the method can work with norm<1?

Reply:
The situation is as follows:
+ Due to floating-point round-off, it's often impractical to make the norm exactly equal to 1.
+ Any deviation from norm=1 is going to distort the world transform. The further the local rotation norm deviates from 1, the greater the distortion will be.

Would "norm approximately equal to 1" be clearer?

Reply:
Yes, I think the word 'approximately equal to' is clearer. Thank you.